tests/container/base/Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM centos:8
+
+ARG GOLANG_VERSION=1.15.7
+ARG PYTHON_VERSION=3.8.7
+
+RUN set -x \
+    && echo 'fastestmirror=True' >> /etc/dnf/dnf.conf \
+    && dnf -y update \
+    && dnf -y install \
+        # Receptor build tools
+        git wget make iproute openssl findutils virtualenv \
+        # Python build requirements
+        gcc zlib-devel bzip2 bzip2-devel readline-devel sqlite \
+        sqlite-devel openssl-devel tk-devel libffi-devel xz-devel \
+    && dnf clean all
+
+# Install specific python version
+RUN set -x \
+    && cd /tmp \
+    && wget -q https://www.python.org/ftp/python/${PYTHON_VERSION}/Python-${PYTHON_VERSION}.tgz \
```

Comment:
as discussed, we can install the dnf module instead of building from source
https://www.server-world.info/en/note?os=CentOS_8&p=python&f=4

```
dnf module -y install python38
```
will have to tweak a few other things to use the right python binary

Reply:
Done :)